apps/authentication-portal/src/main/webapp/identifierauth.jsp

```diff
@@ -39,15 +39,30 @@
     String emailUsernameEnable = application.getInitParameter("EnableEmailUserName");
     Boolean isEmailUsernameEnabled = false;
     String usernameLabel = "username";
+    Boolean isMultiAttributeLoginEnabledInTenant = false;
```

Comment:
false is redundant

Reply:
Fixed with 1591ee73e98c6530a217973e9acc72fb277be5af